core/services/relay/evm/mercury/transmitter.go

```diff
@@ -402,7 +403,11 @@ func (mt *mercuryTransmitter) Transmit(ctx context.Context, reportCtx ocrtypes.R
 		Payload: payload,
 	}
 
-	mt.lggr.Tracew("Transmit enqueue", "req.Payload", req.Payload, "report", report, "reportCtx", reportCtx, "signatures", signatures)
+	ts, err := mt.codec.ObservationTimestampFromReport(report)
+	if err != nil {
+		mt.lggr.Warnw("Failed to get observation timestamp from report", "err", err)
+	}
+	mt.lggr.Debugw("Transmit enqueue", "req.Payload", hexutil.Encode(req.Payload), "report", report, "repts", reportCtx.ReportTimestamp, "signatures", signatures, "observationsTimestamp", ts)
```

Comment:
Will this log line be mangled if we didn't get a real `ts` back? Could exclude that key/val for that case

Reply:
Only time that should happen is with block number report, and will just log zero which is ok